src/mushroom-strategy.ts

```diff
@@ -70,6 +72,26 @@ class MushroomStrategy extends HTMLTemplateElement {
       }
     }
 
+    // Create subviews for each device with config in devices section.
+    const devicesWithSubviews = Helper.strategyOptions.devices ?? {};
```

Comment:
[Card Options](https://github.com/AalianKhan/mushroom-strategy/wiki/Strategy-Options#card-options) allows you to list device ids for card configuration. Instead of creating a new option, it is better to extend CustomCardConfig and add the deviceSubView option to it.